local-h5/discuss.json

```
@@ -1,704 +1,704 @@
+        "title": "{{nickname}}-All Posts"
       }
     }
   },
   "waiting": {
-    "tag_green": "[未开始]",
-    "waiting_des": "开启时间"
+    "tag_green": "[Not Started]",
+    "waiting_des": "Start Time"
   },
   "add_comment_modal": {
-    "placeholder": "请输入...",
-    "apply": "发表",
-    "anonymous_apply": "匿名回复",
-    "remind": "*学生发帖、回帖、回复行为，均算作参与并完成讨论任务",
+    "placeholder": "Please enter...",
+    "apply": "Post",
+    "anonymous_apply": "Reply Anonymously",
+    "remind": "*Student's posts, replies, and responses are all considered participation and completion of the discussion task",
     "handle_apply": {
-      "info1": "内容不能为空",
-      "info2": "内容中含有敏感信息，无法提交。",
+      "info1": "Content cannot be empty",
+      "info2": "Content includes sensitive information and cannot be submitted.",
       "is_reply": {
-        "true": "回复",
-        "false": "回帖"
+        "true": "Reply",
+        "false": "Post"
       }
     }
   },
   "add_point": {
     "handle_add_point": {
-      "title": "提示",
-      "message": "此帖匿名发表后系统将不允许编辑，是否确定匿名发表？",
-      "ok": "确定",
-      "cancel": "取消"
+      "title": "Tip",
+      "message": "Once this post is published anonymously, the system will not allow edits. Are you sure you want to post anonymously?",
+      "ok": "Confirm",
+      "cancel": "Cancel"
     },
     "handle_submit": {
-      "name": "主帖",
-      "info1": "正文中含有敏感信息，无法提交。",
-      "info2": "标题中含有敏感信息，无法提交。"
+      "name": "Main Post",
+      "info1": "The body includes sensitive information and cannot be submitted.",
+      "info2": "The title includes sensitive information and cannot be submitted."
     },
     "mobile_point": {
       "header": {
-        "send": "发帖"
+        "send": "Post"
       },
       "discuss_desc": {
         "more_desc": {
-          "true": "收起",
-          "false": "查看更多"
+          "true": "Collapse",
+          "false": "Read More"
         }
       },
       "content": {
-        "item_title1": "帖子标题",
+        "item_title1": "Post Title",
         "textarea_item": {
-          "placeholder": "请输入帖子标题",
-          "required_message": "标题不能为空",
-          "info1": "标题不能超过100个字符",
-          "info2": "标题不能包含特殊字符（* \\ / : ? \" < > 等）"
+          "placeholder": "Please enter the post title",
+          "required_message": "Title cannot be empty",
+          "info1": "Title cannot exceed 100 characters",
+          "info2": "Title cannot include special characters (* \\ / : ? \" < > etc.)"
         },
-        "item_title2": "发帖内容",
+        "item_title2": "Post Content",
         "draft_editor": {
-          "required_message": "正文不能为空",
-          "placeholder": "请输入发帖内容"
+          "required_message": "Body cannot be empty",
+          "placeholder": "Please enter the post content"
         },
         "setting": {
           "picker": {
-            "title": "允许回复",
-            "ok": "是",
-            "cancel": "否",
-            "span": "允许回复"
+            "title": "Allow Replies",
```

Would it be better if
```suggestion
            "title": "Allow Comments",
```